### PR TITLE
Add changelog for v0.3

### PR DESCRIPTION
**Pull request by auria**

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -1,5 +1,45 @@
...
 
+0.3
+^^^
+Thursday Jan 27 2022
+
+* This version has been tested and requires revisions:
+
+  * to verify ceph-osd units: **cs:ceph-osd-315** (`stable/21.10`, or `charmhub rev 511 or higher`)
+  * to verify ceph-mon units: **cs:ceph-mon-61** (`stable/21.10`, or `charmhub rev 71 or higher`)
+  * to verify nova-compute units: **cs:nova-compute-337** (`stable/21.10`, or `charmhub rev 449 or higher`)
```
Comment (r-f-g) [inline docs/changelog.rst:11]: Actually the nova-compute will not work [1].

---
[1]: https://github.com/canonical/juju-verify/blob/c43dfa4ad7a7f7ca4b5cc1e0cd780692bd83f476/tests/functional/tests/tests.yaml#L5

File: docs/changelog.rst [hunk 2]
```diff
@@ -1,5 +1,45 @@
...
+  * to verify neutron-gateway units: **cs:neutron-gateway** (`stable/21.10`, or `charmhub rev 487`)
+
+* Testing
+
+  * Lint: Add black and isort
+  * Functests: nova-compute and neutron-gateway tests run on the same model. Fix neutron-gateway routers/networks order.
```
Comment (r-f-g) [inline docs/changelog.rst:17]: The nova-compute func tests are commented.

**Review by r-f-g — CHANGES_REQUESTED** [review]

File: docs/changelog.rst (hunk 1 above)
Comment (auria) [reply docs/changelog.rst:11]: This seemed like a problem on a test environment that should be fixed by now. There is still WIP to strengthen the "instance-count" Juju action, but I'm going to try and see if tests pass now in order to leave the NovaCompute integration tests enabled.

**Comment by auria** [comment]
recheck-full

File: docs/changelog.rst (hunk 1 above)
Comment (mkalcok) [reply docs/changelog.rst:11]: As @auria says, the problem was in CI infra, not the nova-compute charm itself. There's a [PR at work](https://review.opendev.org/c/openstack/charm-nova-compute/+/823506) that should make the nova-compute more resilient towards such environment problems.

**Comment by r-f-g** [comment]
> recheck-full

**Review by r-f-g — APPROVED** [review]

**Comment by auria** [comment]
recheck-full

**Pull request merged**